<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>CGeneralSettingsDlgBase</class>
  <widget class="QDialog" name="CGeneralSettingsDlgBase">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>270</width>
        <height>230</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
      </sizepolicy>
    </property>
    <property name="cursor">
      <cursor>0</cursor>
    </property>
    <property name="windowTitle">
      <string>General settings</string>
    </property>
    <property name="windowIcon">
      <pixmap>image0</pixmap>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <widget class="QGroupBox" name="GroupReceiverCoord">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>7</vsizetype>
            </sizepolicy>
          </property>
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>0</height>
            </size>
          </property>
          <property name="title">
            <string>Default Receiver coordinates</string>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="0">
              <layout class="QHBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="TextLatitude">
                    <property name="text">
                      <string>Latitude   </string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLatitudeDegrees">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>30</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>2</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextDegLat">
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                    <property name="text">
                      <string>°</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLatitudeMinutes">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>30</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>2</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextMinutesLat">
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                    <property name="text">
                      <string>'</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLatitudeNS">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>20</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>1</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextNS">
                    <property name="text">
                      <string>N/S</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer10_2">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item row="1" column="0">
              <layout class="QHBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="TextLongitude">
                    <property name="text">
                      <string>Longitude</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLongitudeDegrees">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>30</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>3</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextDegLong">
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                    <property name="text">
                      <string>°</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLongitudeMinutes">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>30</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>2</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextMinutesLong">
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                    <property name="text">
                      <string>'</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="EdtLongitudeEW">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>1</vsizetype>
                      </sizepolicy>
                    </property>
                    <property name="maximumSize">
                      <size>
                        <width>20</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text">
                      <string/>
                    </property>
                    <property name="maxLength">
                      <number>1</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextEW">
                    <property name="text">
                      <string>E/W</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer10">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="GroupGPS">
          <property name="title">
            <string>GPS</string>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="0">
              <layout class="QGridLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item row="0" column="0">
                  <widget class="QCheckBox" name="CheckBoxUseGPS">
                    <property name="text">
                      <string>Use GPS</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="2">
                  <widget class="QLineEdit" name="LineEditGPSPort"/>
                </item>
                <item row="0" column="2">
                  <widget class="QLineEdit" name="LineEditGPSHost"/>
                </item>
                <item row="1" column="3">
                  <spacer name="Spacer6">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="0">
                  <spacer name="Spacer4">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="0" column="3">
                  <spacer name="Spacer5_2">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="0" column="1">
                  <widget class="QLabel" name="GPSHost">
                    <property name="text">
                      <string>Host</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QLabel" name="GPSPort">
                    <property name="text">
                      <string>Port</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer5">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QPushButton" name="buttonOk">
                  <property name="text">
                    <string>&amp;OK</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                  <property name="default">
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <tabstops>
    <tabstop>EdtLatitudeDegrees</tabstop>
    <tabstop>EdtLatitudeMinutes</tabstop>
    <tabstop>EdtLatitudeNS</tabstop>
    <tabstop>EdtLongitudeDegrees</tabstop>
    <tabstop>EdtLongitudeMinutes</tabstop>
    <tabstop>EdtLongitudeEW</tabstop>
    <tabstop>buttonOk</tabstop>
  </tabstops>
  <images>
    <image name="image0">
      <data format="XPM.GZ" length="668">789c75d0cd4ac4301007f07b9f226c6f45bab6ddb45bc447103c0ae22199ccf46337bbdbba1e447c773b49aa28710894dfbf1992cc36134f8f0f22db26af57751d4040af669199376bdf9f5fee3f924d518b6595b7a2d8dc249b5c804877a52cf6c04c03a527382a29db1df312d87a2aa62ca5f19c3ceba275bd4747b5d2044a4fcbacabda28c9d48eba919e1d730f6da90df3c4841ab441263a2290e7792536cc8169102b24e6cc44424dee6f1f489e07263554798e810db95e7274e566f5a7d62ccd95d66af9fc646090bbd0c077065d3f2cd5d3d841c852331e8edcd50da349c33ea425b3d6e61d61d8a788b3139d2f1391f2990ed93ca99ef4ff59ac377646f42e913b47df169dc1ef597dde255fbf0ea69a</data>
    </image>
  </images>
</ui>
